profiles/agung2001.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/4479106/208213842-00ef8a2c-d048-4406-9c66-7569392956c2.png",
+    "issueId": 1565,
+    "name": "Agung Sundoro",
+    "username": "agung2001"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

hi, @Panquesito7 thx for reviewing my PR 😄🙏
I've just [committed a new update](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/1566/commits/5ea9ece929c224068e7d6b8a7510bbe7d069e6a0), I hope it'll fix the issue.
Just let me know if there's more to it. cheers mate 😄🍻